### PR TITLE
add initial version of test-protocol template

### PR DESCRIPTION
**Pull request by tobiscr**
**Description**

Template for the test protocol of ACM, required for Microdeliveries.

**Related issue(s)**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/test-protocol.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+name: Test protocol
+about: Document a manual testing cycle
+---
+
+# Test details
+
+* Tested ACM Module Version: {e.g. 1.1.5}
+* Github release: {Link to GH release}
```
Comment (grego952) [inline .github/ISSUE_TEMPLATE/test-protocol.md:9]:
```suggestion
* GitHub release: {Link to GH release}
```

File: .github/ISSUE_TEMPLATE/test-protocol.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+* Test execution date: {2025-07-29}
+* Tester: {@your-Github-user}
```
Comment (grego952) [inline .github/ISSUE_TEMPLATE/test-protocol.md:11]:
```suggestion
* Tester: {@your-GitHub-user}
```

File: .github/ISSUE_TEMPLATE/test-protocol.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+# Test protocol
+
+The detailed test flow is explained in our internal documentation.
+
+## Pre-requisites
+
+- [ ] Configure System Landscape and Formations in BTP Cockpit
+- [ ] Access to Back Office and Storefront possible
```
Comment (grego952) [inline .github/ISSUE_TEMPLATE/test-protocol.md:20]:
```suggestion
- [ ] Access to back-office and storefront possible
```
based on [this doc](https://doc.oroinc.com/user/concept-guides/administration/menus/) this is how it should be spelled

File: .github/ISSUE_TEMPLATE/test-protocol.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+## Test flow:
+
+### 1. Register events in Back office
```
Comment (grego952) [inline .github/ISSUE_TEMPLATE/test-protocol.md:24]:
```suggestion
### 1. Register events in back-office
```

File: .github/ISSUE_TEMPLATE/test-protocol.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+ - [ ] Passed
+ - [ ] Failed
+ 
+     Link to Github issue: {GH link}
```
Comment (grego952) [inline .github/ISSUE_TEMPLATE/test-protocol.md:29]:
```suggestion
     Link to GitHub issue: {GH link}
```

File: .github/ISSUE_TEMPLATE/test-protocol.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+### 2. Prepare serverless function in Kyma
```
Comment (grego952) [inline .github/ISSUE_TEMPLATE/test-protocol.md:31]:
```suggestion
### 2. Prepare Serverless Function in Kyma
```

File: .github/ISSUE_TEMPLATE/test-protocol.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+
+ - [ ] Passed
+ - [ ] Failed
+
+     Link to Github issue: {GH link}
```
Comment (grego952) [inline .github/ISSUE_TEMPLATE/test-protocol.md:36]:
```suggestion
     Link to GitHub issue: {GH link}
```

File: .github/ISSUE_TEMPLATE/test-protocol.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+
+### 3. Trigger an event in EC
+
+ - [ ] Passed
+ - [ ] Failed
+
+     Link to Github issue: {GH link}
```
Comment (grego952) [inline .github/ISSUE_TEMPLATE/test-protocol.md:43]:
```suggestion
     Link to GitHub issue: {GH link}
```

File: .github/ISSUE_TEMPLATE/test-protocol.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+
+### 3. Verify event was delivered and API called
+
+ - [ ] Passed
+ - [ ] Failed
+
+     Link to Github issue: {GH link}
```
Comment (grego952) [inline .github/ISSUE_TEMPLATE/test-protocol.md:50]:
```suggestion
     Link to GitHub issue: {GH link}
```

File: .github/ISSUE_TEMPLATE/test-protocol.md [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+
+## Cleanup
+
+- [ ] In the Backoffice, delete the created Destination Target and Customer.
```
Comment (grego952) [inline .github/ISSUE_TEMPLATE/test-protocol.md:54]:
```suggestion
- [ ] In back-office, delete the created Destination Target and Customer.
```

**Review by grego952 — APPROVED** [review]

**Comment by VOID404** [comment]
We could one day convert it to a form, for a nicer UX when filling out.

**Pull request merged**